internal/metrics/prometheus.go

```diff
@@ -17,11 +17,20 @@ type CtrlFuncMetrics struct {
 	// ConfigPushCount is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
 	ConfigPushCount *prometheus.CounterVec
 
+	// ConfigPushBrokenResources is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
+	ConfigPushBrokenResources *prometheus.GaugeVec
+
 	// TranslationCount is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
 	TranslationCount *prometheus.CounterVec
 
+	// TranslationBrokenResources is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
+	TranslationBrokenResources prometheus.Gauge
+
 	// ConfigPushDuration is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
 	ConfigPushDuration *prometheus.HistogramVec
+
+	// ConfigPushSuccessTime is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
+	ConfigPushSuccessTime *prometheus.GaugeVec
```

Comment:
Now as I look at it I believe those descriptions don't make much sense for the reader. We already have the help describing the metric registries below so I'm not sure we'd like to drop those here completely or make a comment justifying lack thereof (and probably a `nolint` clause to prevent linter from complaining).

Reply:
I suggest to add more explaining comments here to tell what does the metrics mean.

Reply:
golangci apparently doesn't care, so swapped for a lone comment above indicating where these are actually described.